python/tank/commands/core_localize.py

```diff
@@ -110,36 +109,38 @@ def do_localize(log, pc_root_path, suppress_prompts):
     log.info("")
 
     try:
+        # Step 1: First get a list of all bundle descriptors
+        # key by descriptor repr, which ensures uniqueness
```

Comment:
Nitpick. End your sentence with a period and start the next one with a capital. I had to reread the comment 3 times before realizing it was actually two sentences.